ethportal-peertest/src/jsonrpc.rs

```diff
@@ -28,10 +29,6 @@ const ALL_ENDPOINTS: [JsonRpcEndpoint; 6] = [
         method: "discv5_routingTableInfo",
         id: &2,
     },
-    JsonRpcEndpoint {
-        method: "eth_blockNumber",
-        id: &3,
-    },
     JsonRpcEndpoint {
         method: "portalHistory_dataRadius",
         id: &4,
```

Comment:
Technically these ids are no longer in order, but it doesn't really make a difference.

Reply:
Yeah, these ids seem arbitrary, and maybe not around for a long time, so I won't tinker too much.